packages/patternfly-4/react-core/src/components/Dropdown/examples/Dropdown.md

```diff
@@ -1111,6 +1111,7 @@ A react-router Link may be wrapped by DropdownItem or used directly within Dropd
 
 ```js noLive
 /** Wrapped Link for DropdownItem list **/
+<>
```

Comment:
@ddonahue007 Is it possible to make the eslint check our JSX to ensure we use `<React.Fragment>` instead of `<>`? That would fix some of our fullscreen examples.